docs/book/src/component-config-tutorial/testdata/project/config/rbac/kustomization.yaml

```diff
@@ -10,15 +10,15 @@ resources:
 - leader_election_role.yaml
 - leader_election_role_binding.yaml
 # Comment the following 4 lines if you want to disable
-# the auth proxy (https://github.com/brancz/kube-rbac-proxy)
-# which protects your /metrics endpoint.
-- auth_proxy_service.yaml
-- auth_proxy_role.yaml
-- auth_proxy_role_binding.yaml
-- auth_proxy_client_clusterrole.yaml
+# the metrics which protects your /metrics endpoint.
+- metrics_service.yaml
+- metrics_role.yaml
+- metrics_role_binding.yaml
+- metrics_client_cluster_role.yaml
```

Comment:
I don't think this client clusterrole is needed.

Reply:
I thought those were related to Prometheus but you are right
I checked it we can either simplify here. 
Thank you a lot for pointing out.